app/store/sessionV2.go

```diff
@@ -293,8 +293,8 @@ func (s *SessionV2) MarkRead() error {
 	return err
 }
 
-// GetMoreMessageList loads more messages from before the lastID
-func (s *SessionsV2) GetMoreMessageList(ID int64, lastID string) (error, *MessageList) {
+// GetMoreMessageList loads more messages from before the timestamp sentat
+func (s *SessionsV2) GetMoreMessageList(ID int64, sentat uint64) (error, *MessageList) {
```

Comment:
just a small comment from my side here: I got confused at first as to what this parameter name meant - once I saw sentAt below, I understood the word consisted of two words. I would suggest to use camel casing for the parameter here as well if possible to avoid confusion, if possible.

Reply:
Done.